contracts/examples/crowdsale/SampleCrowdsaleToken.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.11;
+
+
+import "../../token/MintableToken.sol";
+
+
+/**
+ * @title SampleCrowdsaleToken
+ * @dev Very simple ERC20 Token that can be minted.
+ * It is meant to be used in a crowdsale contract.
+ */
+contract SampleCrowdsaleToken is MintableToken {
+
+  string public constant name = "Sample Crowdsale Token";
+  string public constant symbol = "SCT";
+  uint256 public constant decimals = 18;
```

Comment:
`decimals` should be `uint8` according to [ERC20](https://github.com/frozeman/EIPs/blob/94bc4311e889c2c58c561c074be1483f48ac9374/EIPS/eip-20-token-standard.md#decimals). (I've just realized this is wrong in the `SimpleToken` example too.)